ssh/main.go

```diff
@@ -37,105 +36,92 @@ func main() {
 	tunnel := sshTunnel.NewTunnel("/ssh/connection", "/ssh/revdial")
 
 	router := tunnel.GetRouter()
-	router.HandleFunc("/sessions/{uid}/close", func(response http.ResponseWriter, request *http.Request) {
-		exit := func(response http.ResponseWriter, status int, err error) {
-			log.WithError(err).WithFields(log.Fields{
-				"status": status,
-			}).Error("failed to close the session")
+	router.POST("/sessions/:uid/close", func(c echo.Context) error {
```

Comment:
I and @gustavosbarreto have seen it must be `.Any` instead of `.POST`.

Reply:
correct to Any